terraform/modules/elasticache-redis/main.tf

```diff
@@ -0,0 +1,41 @@
+# Pulled from https://github.com/turnerlabs/terraform-aws-elasticache-redis
+resource "aws_security_group" "redis" {
+  vpc_id = "${var.vpc_id}"
+
+  tags {
+    Name          = "${var.tag_name}"
```

Comment:
Can we name things consistently with the rest of the resources so we can identify them. Instead of `tvs2-staging` it would be good to have the resource in the name so: `tvs2-staging-redis-sg`